### PR TITLE
Used mqtts with the azure bridge

### PR DESCRIPTION
**Pull request by AntoineRondelet**

### CODE REVIEW

File: bridge.js [hunk 1]
```diff
@@ -12,7 +13,25 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
 const keyName = process.env.TTN_AZURE_KEYNAME;
 const key = process.env.TTN_AZURE_KEY;
 
-const bridge = new ttnazureiot.Bridge(region, appId, accessKey, hubName, keyName, key);
+var options;
+
+try {
+  const mqttCertPath = process.env.TTN_MQTT_CERT;
```
Comment (johanstokking) [inline bridge.js]: Fallback to hardcoded path; ` || '/etc/ttn/mqtt.ca';`

File: bridge.js [hunk 2]
```diff
@@ -12,7 +13,25 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
...
+  }
+  options = {
+    protocol: 'mqtts',
+    ca: [ fs.readFileSync(mqttCertPath) ],
+  };
+}
+catch(err) {
+  options = {};
+  console.warn('Could not connect using TLS %s', err);
+  console.warn("Fallback to a non-secure connection");
```
Comment (johanstokking) [inline bridge.js]: Never do this, this bloats our logs

File: bridge.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+const fs = require('fs');
 
 const ttnazureiot = require('.');
```
Comment (johanstokking) [inline bridge.js:4]: Keep the `require`s together

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: bridge.js [hunk 4]
```diff
@@ -13,23 +12,11 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
...
-    protocol: 'mqtts',
-    ca: [ fs.readFileSync(mqttCertPath) ],
-  };
-}
-catch(err) {
-  options = {};
-  console.warn('Could not connect using TLS %s', err);
-  console.warn("Fallback to a non-secure connection");
-}
+const mqttCertPath = process.env.TTN_MQTT_CERT || '/etc/ttn/mqtt-ca.pem'
```
Comment (johanstokking) [inline bridge.js]: Add `;`

File: bridge.js [hunk 5]
```diff
@@ -1,6 +1,5 @@
 'use strict';
```
Comment (johanstokking) [inline bridge.js]: + empty line after

File: bridge.js [hunk 6]
```diff
@@ -13,23 +12,11 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
...
-}
-catch(err) {
-  options = {};
-  console.warn('Could not connect using TLS %s', err);
-  console.warn("Fallback to a non-secure connection");
-}
+const mqttCertPath = process.env.TTN_MQTT_CERT || '/etc/ttn/mqtt-ca.pem'
+var options = {
+  protocol: 'mqtts',
+  ca: [ fs.readFileSync(mqttCertPath) ],
```
Comment (johanstokking) [inline bridge.js]: Why an array?

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: bridge.js (hunk 6 above)
Comment (AntoineRondelet) [reply bridge.js]: It can be an array or a string. `ca<string> | <string[]> | <Buffer> | <Buffer[]>`

**Review by johanstokking — APPROVED** [review]

**Pull request merged**